src/style.css

```diff
@@ -0,0 +1,225 @@
+}
+
+.input-big:focus {
+  box-sizing: border-box;
+  position: relative;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.input-big::placeholder {
+  position: relative;
+  left: 0;
+  right: 20px;
+  top: 0;
+  bottom: -16px;
+
+  font-family: "Avenir", sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4d61;
+}
+
+.input-big::placeholder:hover {
+  position: relative;
+  left: 1px;
+  right: 20px;
+  top: 0;
+  bottom: -16px;
+  font-family: "Avenir", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.input-big::placeholder:focus {
+  position: relative;
+  left: 1px;
+  right: 20px;
+  top: 0;
+  bottom: -16px;
+  font-family: "Avenir", sans-serif;
+  font-style: normal;
+  font-weight: 900;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.big-icon {
+  box-sizing: border-box;
+  position: absolute;
+  z-index: 1;
+  width: 19px;
+  height: 19px;
+  left: 34px;
+  top: 45px;
+}
+
+.big-icon:hover {
+  box-sizing: border-box;
+  position: absolute;
+  z-index: 1;
+  width: 19px;
+  height: 19px;
+  left: 34px;
+  top: 45px;
+}
+
+.big-icon:focus {
+  box-sizing: border-box;
+  position: absolute;
+  z-index: 1;
+  width: 19px;
+  height: 19px;
+  left: 34px;
+  top: 45px;
+}
+
+.input-small {
+  width: 100%;
+  height: 42px;
+  padding-left: 33px;
+  box-sizing: border-box;
+  position: relative;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.input-small:hover {
+  width: 100%;
+  height: 42px;
+  box-sizing: border-box;
+  position: relative;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.input-small:focus {
+  width: 100%;
+  height: 42px;
+  box-sizing: border-box;
+  position: relative;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.input-small::placeholder {
+  position: relative;
+  left: 0;
+  right: 6px;
+  top: 0;
+  bottom: 6px;
+
+  font-family: "Avenir", sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+}
+
+.input-small::placeholder:hover {
+  position: relative;
+  left: 40px;
+  right: 6px;
+  top: 0;
+  bottom: 6px;
+  font-family: "Avenir", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+}
+
+.input-small::placeholder:focus {
+  position: relative;
+  left: 40px;
+  right: 6px;
+  top: 0;
+  bottom: 6px;
+  font-family: "Avenir", sans-serif;
+  font-style: bold;
+  font-weight: 900;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+}
+
+.small-icon {
+  height: 11px;
+  width: 11px;
+  box-sizing: border-box;
```

Comment:
Use `box-sizing` rule for the `*` selector:
```
* {
  box-sizing: border-box;
}
```
And remove it everywhere you use it.